content/updates/2018-jan.md

```diff
@@ -0,0 +1,21 @@
+---
+title: January 2018
```

Comment:
I think it's ambitious to call this the January 2018 update when we're now well in to February.